packages/material-ui/src/Table/Table.js

```diff
@@ -67,12 +47,12 @@ Table.propTypes = {
   /**
    * Allows TableCells to inherit padding of the Table.
    */
-  padding: PropTypes.oneOf(['default', 'checkbox', 'dense', 'none']),
+  padding: PropTypes.oneOf(['normal', 'checkbox', 'dense', 'none', 'denseCheckbox']),
```

Comment:
If we follow https://github.com/mui-org/material-ui/issues/13028, we should rename default to something that has more meaning.

Reply:
What does `denseCheckbox` have to do with the table here? :scream:

Reply:
It carries less information. If you'll look at the docs you see that the default value for that prop is...well...`default`. You haven't learned much then.

Reply:
"My button as normal padding by default" vs. "My button has default padding by default"

I still need to know what "default" or "normal" is, regardless of the name. I also need to know what "small" and "large" mean, though, so that's fine for me. :+1: